content/components/overview/PausingRecipes.tsx

```diff
@@ -0,0 +1,119 @@
+
+export const PausingRecipes: React.FC<{
+  data: Array<{ pauseDate: number; recipe: RecipeV3 }>;
+}> = ({ data }) => {
+  const sevenDaysFuture = new Date();
+  sevenDaysFuture.setDate(sevenDaysFuture.getDate() + 7);
+
+  const renderPausingSoonListItem = (): Array<ReactElement> => {
+    const pausingRecipes = data.filter((recipeData) => {
+      const pausingDate = new Date(recipeData.pauseDate);
+      return pausingDate <= sevenDaysFuture;
+    });
+    pausingRecipes.sort((exp1, exp2) => {
+      return exp1.pauseDate - exp2.pauseDate;
+    });
+    return chunkBy(pausingRecipes, 3).map((recipeChunk, rowIdx) => {
+      return (
+        <Row key={rowIdx}>
+          {recipeChunk.map((recipeData, colIdx) => {
+            return (
+              <Col key={`col-${colIdx}`} md={8} sm={24}>
+                <PausingRecipeCard recipeData={recipeData} />
+              </Col>
+            );
+          })}
+        </Row>
+      );
+    });
+  };
+
+  return (
+    <>
+      <h3>Pausing Recipes</h3>
+      {renderPausingSoonListItem()}
+    </>
+  );
+};
+
+const PausingRecipeCard: React.FC<{
+  recipeData: { pauseDate: number; recipe: RecipeV3 };
+}> = ({ recipeData }) => {
+  const { pauseDate, recipe } = recipeData;
+  const [isButtonLoading, setIsButtonLoading] = React.useState(false);
+  const normandyApi = useSelectedNormandyEnvironmentAPI();
+
+  const handlePauseClick = async (): Promise<void> => {
+    setIsButtonLoading(true);
+    try {
+      const updatedData = await normandyApi.patchRecipe(recipe.id, {
+        comment: "One-click pause",
+        arguments: {
+          ...recipe.latest_revision.arguments,
+          isEnrollmentPaused: true,
+        },
+      });
+
+      await normandyApi.requestApproval(updatedData.latest_revision.id);
+    } catch (err) {
+      console.warn(err.message, err.data);
+      Alert.error(`An Error Occurred: ${err.message}`, 5000);
+    } finally {
+      setIsButtonLoading(false);
+    }
+  };
+
+  const pausingDate = new Date(pauseDate);
+
+  return (
+    <Panel
+      bordered
+      className="recipe-listing mb-2"
+      header={<CardHeader recipe={recipe} />}
+    >
+      <div className="d-flex flex-wrap m-0">
+        <div className="flex-grow-1">
+          <p>Experimenter Pause Date: </p>
+          {pausingDate.toLocaleDateString()}
+        </div>
+
+        <IconButton
+          className="ml-1"
+          color="yellow"
+          icon={<Icon icon="close-circle" />}
+          loading={isButtonLoading}
+          onClick={handlePauseClick}
+        >
+          Pause
+        </IconButton>
+      </div>
+    </Panel>
+  );
+};
+
+const CardHeader: React.FC<{ recipe: RecipeV3 }> = ({
+  recipe,
+}): ReactElement => {
+  return (
+    <>
+      <Link to={`recipes/${recipe.id}`}>
+        <Tag className="mr-half" color="violet">
+          {recipe.id}
+        </Tag>
+      </Link>
+      <Link
+        style={{ textDecoration: "inherit", color: "inherit" }}
```

Comment:
ohhh, yes i was gonna highlight this when I initially had this PR up for review...
I was trying to get rid of the ugly blue and underlining, probs not the way to go about it

Reply:
Oh I meant a comment in the code, so we aren't confused in the future.. :) I don't have any ideas for how to get rid of it either. May @rdalal does, but I think with a comment added here it will be fine.

Reply:
my preference for this kind of thing would be to create a class that can be re-used places we want to do this. maybe something like `.link-inherit-color-decoration`.